package.json

```diff
@@ -21,8 +21,8 @@
   "main": "./dist/js/okta-sign-in.entry.js",
   "scripts": {
     "clean": "rimraf target && rimraf dist && rimraf build2",
-    "lint:report": "grunt lint && node ./buildtools/stylelint/stylelint.js checkstyle || true && eslint -f checkstyle -o build2/OSW-eslint-checkstyle.xml . || true",
-    "lint": "grunt lint && node ./buildtools/stylelint/stylelint.js && eslint .",
+    "lint:report": "grunt lint && stylelint assets/sass/ > build2/OSW-stylelint-checkstyle.xml --custom-formatter node_modules/stylelint-checkstyle-formatter || true && eslint -f checkstyle -o build2/OSW-eslint-checkstyle.xml . || true",
```

Comment:
Couldn’t find a better way to use the `stylelint-checkstyle-formatter` from the cli, do you guys have a better idea or you think Is fine to reference a node_modules dir?

Reply:
This is probably OK for now. In the future we could consider extracting all `lint` scripts into a new task runner, similar to what we do in [courage](https://github.com/okta/okta-ui/blob/master/packages/courage/buildtools/commands/lint.js).